gsk4/src/auto/transform.rs

```diff
@@ -50,6 +50,32 @@ impl Transform {
         }
     }
 
+    #[cfg(feature = "v4_20")]
+    #[cfg_attr(docsrs, doc(cfg(feature = "v4_20")))]
+    #[doc(alias = "gsk_transform_matrix_2d")]
+    #[must_use]
+    pub fn matrix_2d(
```

Comment:
Probably needs manual impl similar to other transform methods

Reply:
Done